tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -507,3 +507,50 @@ def test_simple(self):
             (1541066400, [{'count': 1}]),
             (1541070000, [{'count': 2}]),
         ]
+
+
+class OrganizationEventsMetaEndpoint(OrganizationEventsTestBase):
+    def test_simple(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project()
+        project2 = self.create_project()
+        group = self.create_group(project=project)
+        group2 = self.create_group(project=project2)
+        self.create_event('a' * 32, group=group, datetime=self.min_ago)
+        self.create_event('b' * 32, group=group2, datetime=self.min_ago)
```

Comment:
I found an id that will be in-sample at the 10% sample rate. A test using the id: `'m' * 32` should return a count in turbo mode. Admittedly a kind of fragile way to test it though.

```
SELECT cityHash64(toString('mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm')) / pow(2, 64) AS sample_expr

┌─────────sample_expr─┐
│ 0.08591816781397661 │
└─────────────────────┘
```